projects/simple-if-else-dictionary/app.py

```diff
@@ -0,0 +1,30 @@
+#Simple if else Dictionary.
+import json
+from difflib import get_close_matches	#if you enter this library will check it.
+data = json.load(open("data.json"))
+
+def translate(word):
+    word = word.lower()
+    if word in data:
+        return data[word]
+    elif len(word) == 0:
+        return "Please!! Enter word . . . "
+    elif len(get_close_matches(word, data.keys())) > 0:
+        yn = input("Did you mean %s instead?\nEnter Y if yes, or N if no : " % get_close_matches(word, data.keys())[0])
+        if (yn == "Y") or (yn == "y"):
```

Comment:
instead of writing two times use lower or uppercase examples
```suggestion
        if (yn.lower()== "y"):
```